public/js/actions/breakpoints.js

```diff
@@ -1,21 +1,29 @@
+// @flow
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-"use strict";
 
 const constants = require("../constants");
 const { PROMISE } = require("../util/redux/middleware/promise");
 const { getBreakpoint, getBreakpoints } = require("../selectors");
 const { Task } = require("../util/task");
 const fromJS = require("../util/fromJS");
 
-function enableBreakpoint(location) {
+import type { Location } from "./types";
```

Comment:
Is it possible to use commonjs here? or is there something about types that means we need to use import?